app/controllers/vendors_controller.rb

```diff
@@ -5,5 +5,6 @@ def index
 
   def show
     @vendor = Vendor.find(params[:id])
+    @reviews = Review.where(vendor_id: @vendor)
```

Comment:
If you have your associations set up right, you can also call `@vendor.reviews` to accomplish the same thing - that's preferred!